core/consensus/grandpa/impl/grandpa_impl.hpp

```diff
@@ -101,6 +103,13 @@ namespace kagome::consensus::grandpa {
     std::shared_ptr<boost::asio::io_context> io_context_;
     std::shared_ptr<authority::AuthorityManager> authority_manager_;
 
+    bool isReady_ = false;
```

Comment:
better to follow our conventions and name variable `is_ready_`

Reply:
Ok, agree